packages/auth/src/sessionStore.ts

```diff
@@ -93,17 +96,22 @@ export class LocalStorageStore extends SessionDataStore {
       throw new NoSessionDataError('No session data was found in localStorage');
     }
     const dataJSON = JSON.parse(data);
-    return SessionData.fromJSON(dataJSON);
+    const session = SessionData.fromJSON(dataJSON);
+    Object.assign(this.sessionData, session);
```

Comment:
Is it necessary to do this object assign here? Seem unusual that this function would have mutation/side effects.

Reply:
The objective is to keep the same reference to make it a shared variable as similar to `InstanceDataStore`. So that between the parallel calls to deleteFile should be able to get same reference to session variable. 
I also mentioned the `testing` script that proves the correct behaviour. 
cc: @kyranjamie